index/src/transactors/evm_transactor.rs

```diff
@@ -0,0 +1,96 @@
+use crate::traits::Error;
+use alloc::string::String;
+use pink_web3::api::{Eth, Namespace};
+use pink_web3::contract::tokens::Tokenize;
+use pink_web3::contract::{Contract, Options};
+use pink_web3::ethabi::{Bytes, Token, Uint};
+use pink_web3::keys::pink::KeyPair;
+use pink_web3::signing::Key;
+use pink_web3::transports::{resolve_ready, PinkHttp};
+use pink_web3::types::{Res, H160, H256};
+
+/// The client to submit transaction to the Evm evm_contract contract
+pub struct EvmContractClient {
```

Comment:
Looks like the struct is implemented for ChainBridge::Evm2Phala, better to rename.

Reply:
will do

Reply:
I renamed it to `ChainBridgeClient`, let's see if the code is reusable to other bridges, if not, I will remove it and combine it into the logic of the EVM2Phala executor

Reply:
We'd better keep the bridge executor as a one-way executor, my consideration is that we will potentially enable/disable a one-way bridge from the registry if the bridge got a problem in that way. Disabling a one-way bridge, will give the engine information that can not produce a solution edge for that bridge direction.